mindsdb/integrations/handlers/dummy_ml_handler/__init__.py

```diff
@@ -0,0 +1,77 @@
+from select import select
```

Comment:
I'm tempted to keep this one for illustration purposes @StpMax, would you agree? Or can we just remove it entirely.

Reply:
i think better will be use real ml framework for illustration